### ISSUE TITLE
Invalid (X)HTML in case of \mscfile, \dotfile, \diafile

### ISSUE THREAD

**Issue by albert-github** [doxygen/doxygen#7087]
In case of the usage of the commands `\dotfile`, `\mscfile`, `\diafile` the resulting html is invalid (as shown by the xmllint command together with the default doxygen dtd):

```
xmllint --path .../testing/dtd --noout  --nonet --postvalid html/*html
Element div is not declared in p list of possible children
Document html/aa_8h.html does not validate
```

The example is with `\mscfile` but the problem also occurs with `\dotfile` and `\diafile`: [msc_5.zip](https://github.com/doxygen/doxygen/files/3339225/msc_5.zip)

**Comment by albert-github** [comment]
I've just pushed a proposed patch, pull request #9968

**Comment by albert-github** [comment]
Code has been integrated in master on GitHub (please don't close the issue as this will be done at the moment of an official release).

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.9.7.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).
